editor/editor_settings.h

```diff
@@ -36,6 +36,8 @@
 #include "core/io/resource.h"
 #include "core/os/thread_safe.h"
 #include "core/templates/rb_set.h"
+#include "editor/plugins/editor_debugger_plugin.h"
+#include "editor/plugins/editor_plugin.h"
```

Comment:
It looks like it is these includes that break things, as they include `Window` and are now introduced in `lightmapper_rd`, as it included platform things, could `#undef` them after the include there, not sure though